src/dotnet/Users.Service/UserProfilesBackend.cs

```diff
@@ -39,6 +40,29 @@ public UserProfilesBackend(IServiceProvider services) : base(services)
         return await ToUserProfile(user, cancellationToken).ConfigureAwait(false);
     }
 
+
+
+    // [CommandHandler]
+    public virtual async Task UpdateStatus(
+        IUserProfilesBackend.UpdateStatusCommand command,
+        CancellationToken cancellationToken = default)
+    {
+        if (Computed.IsInvalidating()) {
+            _ = AuthBackend.GetUser(command.UserId, default);
+            return;
+        }
+
+        var context = await CreateCommandDbContext(cancellationToken).ConfigureAwait(false);
+        await using var __ = context.ConfigureAwait(false);
+
+        var dbUser = await context.Users.FindAsync(new object?[] { command.UserId }, cancellationToken)
+                .ConfigureAwait(false)
+            ?? throw new KeyNotFoundException($"User with id='{command.UserId}' not found");
+        dbUser.Status = command.NewStatus;
+        context.Users.Update(dbUser);
```

Comment:
You don't need this `Update` call - the entity is already being tracked after `FindAsync`

Reply:
done